pkgs/by-name/pa/pam_fprint_grosshack/package.nix

```
@@ -0,0 +1,64 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  pkg-config,
+  meson,
+  ninja,
+  glib,
+  libfprint,
+  polkit,
+  systemd,
+  pam,
+  libpam-wrapper,
+  dbus,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+
+  pname = "pam_fprint_grosshack";
+  version = "0.3.0";
+
+  src = fetchFromGitLab {
+    owner = "mishakmak";
+    repo = "pam-fprint-grosshack";
+    tag = "v${finalAttrs.version}";
+    sha256 = "sha256-obczZbf/oH4xGaVvp3y3ZyDdYhZnxlCWvL0irgEYIi0=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+  ];
+  buildInputs = [
+    glib
+    dbus
+    libfprint
+    polkit
+    systemd
+    pam
+    libpam-wrapper
+  ];
+
+  mesonFlags = [
+    "-Dpam_modules_dir=${placeholder "out"}/lib/security"
+    "-Dsysconfdir=${placeholder "out"}/etc"
+    "-Ddbus_service_dir=${placeholder "out"}/share/dbus-1/system-services"
+    "-Dsystemd_system_unit_dir=${placeholder "out"}/lib/systemd/system"
+  ];
+
+  env = {
+    PKG_CONFIG_DBUS_1_INTERFACES_DIR = "${placeholder "out"}/share/dbus-1/interfaces";
+    PKG_CONFIG_POLKIT_GOBJECT_1_POLICYDIR = "${placeholder "out"}/share/polkit-1/actions";
+    PKG_CONFIG_DBUS_1_DATADIR = "${placeholder "out"}/share";
+  };
+
```

can be tested with `nix-update pam_fprint_grosshack`
```suggestion

  passthru.updateScript = nix-update-script { };

```

---

It appears nix-update chose to swap out the version for 0.1.0 instead of 0.3.0; how do I control this behavior?
`Update 0.3.0 -> 0.1.0 in /home/agvantibo/Downloads/Builds/nixpkgs/pkgs/by-name/pa/pam_fprint_grosshack/package.nix`

---

nix-update doesn't work correctly on gitlab when the project doesn't have a release for each tag, we should leave out the update script for this package.